### DISCUSSION TITLE
Sorting Values in Dataframe

### DISCUSSION THREAD

**Discussion by NeelaGitH** [dsc-tiu/MLStudyJams2022#12] [Q&A] [answered]
So I was doing the Kaggle exercise, and this was the question where it asked to sort the Series based on values and we use the sort_values() function for sorting based on values. But when I submitted the answer, it's showing it was wrong and the correct function is sort_index(). But as far as I know, this function is used for sorting based on indexes. So I can't understand how it's the correct answer. 
![image](https://user-images.githubusercontent.com/119100648/232337878-a6243b21-2c8f-48a5-aac0-6cd01bce1eae.png)
![image](https://user-images.githubusercontent.com/119100648/232337921-308f8637-8dda-4db3-a539-48270626fa79.png)

**Comment by diffrxction** [comment, marked as answer]
Hey @NeelaGitH sorry for the late response!

Here, it appears you were asked to firstly sort the values by the index associated with price which is why the `sort_index()` method is correct. If you were using `sort_values()` that didn't work mostly because you had to sort one specific index in ascending order and the `sort_values()` function does not do so as intended.

To further understand my point I would recommend trying to download this dataset into your local system and printing out the two results when using `sort_values()` and `sort_index()` respectively.
